float32 random_number

